src/main/kotlin/com/pk/Main.kt

```diff
@@ -43,9 +43,9 @@ class MyArgs(parser: ArgParser) {
         }
     }.default<List<Dependency>>(listOf())
 
-    val useArgParser by parser.flagging(
-        "-p", "--parseArgs",
-        help = "set up command-line arg parsing"
+    val noArgs by parser.flagging(
+        "-n", "--no-arg-parsing",
```

Comment:
Just FYI another way to do this while keeping your original code would have been to leave the parseArgs flag as is and just default it to true here. That's what I was imagining, but flipping the logic works too.